backend/app/services/search.py

```diff
@@ -9,6 +9,17 @@
 class ElasticService:
     # todo add bulkhead pattern
     # todo add error message for unauthorized user
+    # Define the keywords for sorting.
+    DIRECTIONS = ("asc", "desc")
+    FIELDS = (
+        "ciSystem",
+        "releaseStream",
+        "platform",
+        "workerNodesCount",
+        "masterNodesCount" "infraNodesCount",
+        "totalNodesCount",
+    )
+
```

Comment:
Are either of these actually *used* anywhere?

Reply:
Not used. So removed them